tests/test_plugin.py

```diff
@@ -0,0 +1,216 @@
+#! python3  # noqa E265
+
+"""
+    Test for the plugin class (subclass of mkdocs.BasePlugin).
+
+    Usage from the repo root folder:
+
+        # all tests
+        python -m unittest tests.test_plugin
+
+        # specific test
+        python -m unittest tests.test_plugin.TestMkdocsPlugin.
+"""
+
+# #############################################################################
+# ########## Libraries #############
+# ##################################
+
+# Standard library
+from pathlib import Path
+import logging
+import unittest
+
+# MkDocs
+from mkdocs.config import load_config
+
+# package
+from mkdocs_git_revision_date_localized_plugin.plugin import (
+    GitRevisionDateLocalizedPlugin,
+)
+
+
+# #############################################################################
+# ######## Globals #################
+# ##################################
+
+# make this test module easily reusable
+PLUGIN_NAME = "git-revision-date-localized"
+
+# custom log level to get plugin info messages
+logging.basicConfig(level=logging.INFO)
+
+# #############################################################################
+# ########## Helpers ###############
+# ##################################
+
+
+# #############################################################################
+# ########## Classes ###############
+# ##################################
+
+
+class TestMkdocsPlugin(unittest.TestCase):
```

Comment:
You are using `unittest` instead of `pytest`. Next time let's standardize on `pytest`. No need to rewrite for now, thanks for extending test suite!

Reply:
Unittest is included in PSL and can also be run with pytest, as it's done in CI Github workflow.